<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="4" total="6" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Test" duration-ms="61666" started-at="2020-06-02T08:05:46Z" finished-at="2020-06-02T08:06:47Z">
    <groups>
    </groups>
    <test name="exampletest" duration-ms="61666" started-at="2020-06-02T08:05:46Z" finished-at="2020-06-02T08:06:47Z">
      <class name="test.chipViewTest">
        <test-method status="PASS" signature="ExtentSetup()[pri:0, instance:test.chipViewTest@7364985f]" name="ExtentSetup" is-config="true" duration-ms="87" started-at="2020-06-02T08:05:46Z" finished-at="2020-06-02T08:05:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentSetup -->
        <test-method status="PASS" signature="setup()[pri:0, instance:test.chipViewTest@7364985f]" name="setup" is-config="true" duration-ms="37465" started-at="2020-06-02T08:05:46Z" finished-at="2020-06-02T08:06:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="clickChipView()[pri:0, instance:test.chipViewTest@7364985f]" name="clickChipView" duration-ms="18690" started-at="2020-06-02T08:06:23Z" finished-at="2020-06-02T08:06:42Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at test.chipViewTest.clickChipView(chipViewTest.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickChipView -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:test.chipViewTest@7364985f]" name="getResult" is-config="true" duration-ms="893" started-at="2020-06-02T08:06:42Z" finished-at="2020-06-02T08:06:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=clickChipView status=FAILURE method=chipViewTest.clickChipView()[pri:0, instance:test.chipViewTest@7364985f] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.chipViewTest@7364985f]" name="tearDown" is-config="true" duration-ms="0" started-at="2020-06-02T08:06:47Z" finished-at="2020-06-02T08:06:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="ExtentTearDown()[pri:0, instance:test.chipViewTest@7364985f]" name="ExtentTearDown" is-config="true" duration-ms="3583" started-at="2020-06-02T08:06:48Z" finished-at="2020-06-02T08:06:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTearDown -->
      </class> <!-- test.chipViewTest -->
      <class name="test.DownloadAppTest">
        <test-method status="FAIL" signature="UpdateAppPositive()[pri:0, instance:test.DownloadAppTest@5d20e46]" name="UpdateAppPositive" duration-ms="2286" started-at="2020-06-02T08:06:43Z" finished-at="2020-06-02T08:06:46Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at test.DownloadAppTest.UpdateAppPositive(DownloadAppTest.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- UpdateAppPositive -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:test.DownloadAppTest@5d20e46]" name="getResult" is-config="true" duration-ms="1491" started-at="2020-06-02T08:06:46Z" finished-at="2020-06-02T08:06:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=UpdateAppPositive status=FAILURE method=DownloadAppTest.UpdateAppPositive()[pri:0, instance:test.DownloadAppTest@5d20e46] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
      </class> <!-- test.DownloadAppTest -->
    </test> <!-- exampletest -->
  </suite> <!-- Test -->
</testng-results>
